src/system/settings.ts

```diff
@@ -94,22 +102,22 @@ export function registerSystemSettings() {
         scope: 'client',
         config: true,
         type: Number,
-        default: 0 as number,
+        default: TargetingOptions.SelectedOnly as number,
         requiresReload: true,
         choices: {
-            0: game.i18n!.localize(
+            [TargetingOptions.SelectedOnly]: game.i18n!.localize(
                 `SETTINGS.${SETTINGS.APPLY_BUTTONS_TO}.choices.0`,
```

Comment:
Should we also apply the names to the localisation keys?

Reply:
Fair point